go.mod

```diff
@@ -10,14 +10,14 @@ go 1.22.8
 // merge of any upgrades we should communicate to all teams to be on the lookout
 // for behavior changes, just like we would after a go upgrade.
 require (
-	golang.org/x/crypto v0.26.0
+	golang.org/x/crypto v0.27.0
```

Comment:
argh, note the above comment. these core dep upgrades need to be in their own commit. I wonder if we'll get these upgrades for free when dev inf upgrades to go 1.23. Maybe ask dev inf on their timeline for that?

Reply:
oh, very good eye. thanks for noticing!